src/room/RoomPage.tsx

```diff
@@ -82,82 +85,124 @@ export const RoomPage: FC = () => {
     if (optInAnalytics === null && setOptInAnalytics) setOptInAnalytics(true);
   }, [optInAnalytics, setOptInAnalytics]);
 
-  const groupCallView = useCallback(
-    (groupCallState: GroupCallStatus): JSX.Element => {
-      switch (groupCallState.kind) {
-        case "loaded":
+  const wasInWaitForInviteState = useRef<boolean>(false);
+
+  useEffect(() => {
+    if (groupCallState.kind === "loaded" && wasInWaitForInviteState) {
+      logger.log("Play join sound 'Not yet implemented'");
+    }
+  }, [groupCallState.kind]);
+
+  const groupCallView = useCallback((): JSX.Element => {
+    switch (groupCallState.kind) {
+      case "loaded":
+        return (
+          <GroupCallView
+            client={client!}
+            rtcSession={groupCallState.rtcSession}
+            isPasswordlessUser={passwordlessUser}
+            confineToRoom={confineToRoom}
+            preload={preload}
+            skipLobby={skipLobby || wasInWaitForInviteState.current}
+            hideHeader={hideHeader}
+            muteStates={muteStates}
+          />
+        );
+      case "waitForInvite":
+      case "canKnock": {
+        wasInWaitForInviteState.current =
+          wasInWaitForInviteState.current ||
+          "waitForInvite" === groupCallState.kind;
```

Comment:
Nitpick, for consistency with the other `===`s
```suggestion
          groupCallState.kind === "waitForInvite";
```